packages/shared_preferences/shared_preferences_macos/pubspec.yaml

```diff
@@ -2,23 +2,26 @@ name: shared_preferences_macos
 description: macOS implementation of the shared_preferences plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/shared_preferences/shared_preferences_macos
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+shared_preferences%22
-version: 2.0.2
+version: 2.0.3
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=2.0.0"
+  flutter: ">=2.5.0"
 
 flutter:
   plugin:
     implements: shared_preferences
     platforms:
       macos:
         pluginClass: SharedPreferencesPlugin
+        dartPluginClass: SharedPreferencesMacOS
 
 dependencies:
   flutter:
     sdk: flutter
   shared_preferences_platform_interface: ^2.0.0
 
 dev_dependencies:
+  flutter_test:
+    sdk: flutter
   pedantic: ^1.10.0
```

Comment:
Optional: Could you remove this while you are here? It's cruft that I missed when cleaning up analysis options.